src/CoreWCF.Http/tests/SimpleWSHTTPTest.cs

```diff
@@ -73,6 +73,38 @@ public void WSHttpRequestReplyEchoStringTransportSecurity()
             }
         }
 
+        [Fact , Description("Demuxer-failure")]
+        public void WSHttpRequestReplyWithTransportMessageEchoStringDemuxFailure()
+        {
+            ServicePointManager.ServerCertificateValidationCallback += new RemoteCertificateValidationCallback(ValidateCertificate);
+            string testString = new string('a', 3000);
```

Comment:
This shouldn't be needed. It only does something on .NET Framework and you've wired up the X509ServiceCertificateAuthentication lower down anyway. You can fix this in another PR after as it's redundant and doesn't affect anything.